mkdocs_include_markdown_plugin/event.py

```diff
@@ -2,6 +2,8 @@
 import re
 from pathlib import Path
 
+from . import process
```

Comment:
I'm not a fan of relative imports, could you replace with a absolute one to maintain the consistency with `plugin.py` module?

```suggestion
from mkdocs_include_markdown_plugin import process
```